src/javascript/_common/localize.js

```diff
@@ -1,4 +1,5 @@
 const moment   = require('moment');
+require('moment/min/locales');
```

Comment:
In SmartTrader we use `Moment.js` library for handling with Date. I found out that there is a method in it for translating dates into different language - `moment.locale`, bur for proper work it requires importing available languages for library.